README.md

```diff
@@ -37,7 +37,7 @@ DescribedFeatureOverlay(
   featureId: 'featureId1', // unique id that identifies this overlay
   tapTarget: icon,
   title: 'Just how you want it',
-  description: 'Tap the menu icon to switch accounts, change settings & more.',
+  description: Text('Tap the menu icon to switch accounts, change settings & more.'),
```

Comment:
Maybe discard this as #14 will override this anyway?

Reply:
Oh you're right... Now how do I do that 😅?